test/e2e/pages/TaskPage.js

```diff
@@ -8,13 +8,37 @@ class TaskPage {
 
   get taskDescription() { return $('//div[@data-placeholder="Description or type \'/\' for commands"]');}
 
-  get closeWindows() { return $('//div[@cutooltip="Close window"]');}
+  get closeTaskModal() { return $('//div[@cutooltip="Close window"]');}
 
   get taskSettingsButton() { return $('.cu-task-header__control');}
 
   get deleteButton() { return $('.nav-menu-item__icon_remove');}
 
-  getTaskTitle(title) { return $(`//span/div[text()='${title}']`);}
+  get floatingNameInput() { return $('.cu-form__input'); }
+
+  get floatingStatusButton() { return $('.lv-task__status'); }
+
+  get floatingDecriptionInput() { return $('.ql-editor'); }
+
+  get setPriorityButton() { return $('.cu-priorities-view__icon-select'); }
```

Comment:
If it is only a button, maybe the name would be "priorityButton"

Reply:
The button has a "Set priority" tooltip
# 
![New Bitmap Image](https://user-images.githubusercontent.com/41970655/72997373-a6422280-3dd2-11ea-8a1f-7c1929d28745.jpg)